cmd/otelcorecol/builder-config.yaml

```diff
@@ -28,5 +28,6 @@ processors:
 
 replaces:
   - go.opentelemetry.io/collector => ../../
-  - go.opentelemetry.io/collector/semconv => ../../semconv
```

Comment:
Should you not have a change around the exporter module at line 16?

Reply:
Good point, updated.

Reply:
Same thing in the default config in the builder.

Reply:
i forgot to run `make ocb`. should probably add a check for this